tests/math/test_functions.py

```diff
@@ -2466,24 +2466,6 @@ class TestSize:
         ([[0], [1], [2], [3], [4], [5]], 6),
     ]
 
-    @pytest.mark.autograd

```

Comment:
Why are these getting deleted? Is this a merge conflict gone wrong?

Reply:
Nope, but the feature was only added for torch. These tests are meaningless